### PR TITLE
Feature/improved admin flow

### PR DESCRIPTION
**Pull request by klaseskilson**
This introduces some minor changes to where admins get redirected and the links of the drinks form.

New drink -> new drink
Edit drink -> edit drink

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -20,11 +20,8 @@
 
 <%= render 'layouts/header' %>
 
-<% if (defined?(notice) and !notice.blank?) or (defined?(alert) and !alert.blank?) %>
```
Comment (danielronnkvist) [inline app/views/layouts/application.html.erb]: Why do check that it's both defined and not blank? Isn't not blank enough?

Comment (klaseskilson) [inline app/views/layouts/application.html.erb]: That line is beeing removed.

Comment (danielronnkvist) [inline app/views/layouts/application.html.erb]: Lol, true.

**Comment by klaseskilson** [comment]
@danielronnkvist, is this mergable?

**Comment by CarlEnglund** [comment]
Yes i'd say so. @danielronnkvist? :beers:

**Comment by danielronnkvist** [comment]
Yes! Sorry about the delayed answer 🍾<🍺

tis 3 nov. 2015 kl. 13:14 skrev Carl Englund <notifications@github.com>:

**Pull request merged**